### PR TITLE
Add tendermint activation to postman collection

### PR DESCRIPTION
**Pull request by smk762**
Closes https://github.com/KomodoPlatform/komodo-docs-mdx/issues/311
Added examples and note in doc for [breaking change](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2173) at next release

### CODE REVIEW

File: src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx [hunk 1]
```diff
@@ -29,6 +29,10 @@ Use this method to activate Tendermint coins (COSMOS/IRIS/OSMOSIS) and IBC asset
 | tokens\_balances | array of objects | Only returned when `get_balances` is `true`. A list of standard [AddressInfo](/komodo-defi-framework/api/common_structures/#address-info) objects, one for each token. |
 | tokens\_tickers  | array            | Only returned when `get_balances` is `false`. A list of each token which was activated.                                                                                |
 
+<Note>
+ The `rpc_urls` parameter will be replaced with the `nodes` parameter in the next release. Instead of a list of url strings, the `nodes` parameter will be a list of [EvmNode objects](/komodo-defi-framework/api/common_structures/activation/#evm-node).
```
Comment (onur-ozkan) [inline src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx]: Was the "EvmNode objects" part intentional here?

Comment (smk762) [reply src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx]: I've duplicated the `EvmNode` object and named it `TendermintNode` in common structures for better clarity.

Comment (onur-ozkan) [reply src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx]: `gui_auth` is no longer used. We renamed it as `komodo_proxy`

Comment (smk762) [reply src/pages/komodo-defi-framework/api/v20/enable_tendermint_with_assets/index.mdx]: I've updated this across the docs & postman, with a note `Prior to the v2.2.0 release, the "gui_auth" field was used to access RPC nodes run officially by the Komodo Platform team. This field was replaced with the "komodo_auth" field in the v2.2.0 release.`

File: src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx [hunk 2]
```diff
@@ -144,16 +144,68 @@ Contains information electrum servers for coins being used in `Electrum` or `Lig
...
+</CollapsibleSection>
+
+### TendermintNode
+
+The `TendermintNode` object includes the following items for a given coin or token:
+
+| Parameter    | Type    | Description                                                                                                         |
+| ------------ | ------- | ------------------------------------------------------------------------------------------------------------------- |
+| url          | string  | URL of an RPC node                                                                                                  |
+| komodo\_auth | boolean | Optional, defaults to `false`. Must be set to `true` to access RPC nodes run officially by the Komodo Platform team |
```
Comment (onur-ozkan) [inline src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx:185]: I would update this with something like "Must be set to `true` to access RPC nodes run behind [komodo-defi-proxy](https://github.com/KomodoPlatform/komodo-defi-proxy)"

**Review by onur-ozkan — APPROVED** [review]
LGTM other than one note

File: src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx (hunk 2 above)
Comment (smk762) [reply src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx]: done, thanks.

**Review by ShantanuSharma9873 — APPROVED** [review]

**Review by gcharang — APPROVED** [review]

**Pull request merged**